frontend/src/app/edit-claim/edit-claim.component.html

```diff
@@ -1,4 +1,4 @@
-<div *ngIf="claim; else loading">
+<div *ngIf="claim; else loading" class="mbh">
```

Comment:
Overall, I think this needs a little padding between the data and the paper containing it. All of the words/buttons are right on the edge of the paper(s)